packages/markdown/components/Callout/index.jsx

```diff
@@ -9,11 +9,14 @@ const Callout = props => {
    * hast-util's hProps and Slate's MDAST serializer
    */
   const { theme, title, icon } = props;
-
+  const titleProps = children[0].props; // @rafegoldberg sucks
```

Comment:
May wanna do an additional existence check here: 
```
const titleProps = children.length && children[0].props;
```

Reply:
It shouldn't matter, at least in theory. (Since a callout will _always_ have at least an emoji prefix, it's `children` list will similarly always contain at least one sub-component!) That said, it definitely can't hurt...

```suggestion
  const titleProps = children.length && [0].props; // @rafegoldberg sucks
```